### ISSUE TITLE
Purpose of the DAO

### ISSUE THREAD

**Issue by rethore** [ethereum-cph/DAO#1]
I guess the first step to create a DAO would be to identify a purpose / mission. This will drive the design of the organisation, functionalities, smart contracts... I propose to brainstorm here on what our purpose could be.

**Comment by rethore** [comment]
I will start. Our purpose could be:
- To learn/teach how Ethereum is working, and how to develop on it
- To help spreading the word about decentralisation, blockchain tech and Ethereum in Denmark
- To be an incubator for startup-DAOs in Denmark
- To be a consulting DAO for companies in Denmark interested in using Ethereum
- To be a lobby group to influence political decisions on the legal status of DAOs in Denmark
- Networking on Ethereum and drinking some nice beers

**Comment by czepluch** [comment]
+1

**Comment by czepluch** [comment]
I think it would be great to use this as a learning case. We will hopefully be able to build something nice and useful that we can then deploy to the live blockchain. In my mind your first point and last point are the most important things of this project.

**Comment by rethore** [comment]
Yes, I agree. Also the other points would have to be built on those two points. Let's focus on the low hanging fruits first.

**Comment by czepluch** [comment]
https://blog.slock.it/daos-or-how-to-replace-both-the-kickstarter-and-token-presale-models-1b2b8898d6e7#.jfhs46amr

Very very interesting what the guys at slock.it are doing.

**Comment by rethore** [comment]
Yes, I've been following this development with great interest as well. We are living exciting times.
